applications/luci-app-watchcat/root/etc/uci-defaults/40_luci-watchcat

```diff
@@ -1,10 +0,0 @@
-#!/bin/sh
```

Comment:
Sorry if I ask again. Are you sure we don't need this file? As far as I can see, the service has not yet been switched to procd. To trigger an restart or reload on config change.
Or is this created automatically now.
I am unfortunately no longer quite sure how it is now

Reply:
Hey, you were right to point that out.  I needed to make some changes to get it working with ucitrack.  https://github.com/openwrt/packages/pull/14569

Reply:
I would prefer if we could switch the service to procd. Then we would no longer have the annoying ucitrack.

Reply:
Migration to procd has been done